src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java

```diff
@@ -441,7 +441,7 @@ public void testNoFlagsUnSupportedOS() throws ParseException {
         Assert.assertTrue(targets.contains("OSP"));
         Assert.assertTrue(targets.contains("Convert2RHEL"));
         // Complexity
-        Assert.assertEquals(WorkloadInventoryReportModel.COMPLEXITY_MEDIUM,workloadInventoryReportModel.getComplexity());
+        //Assert.assertEquals(WorkloadInventoryReportModel.COMPLEXITY_MEDIUM,workloadInventoryReportModel.getComplexity());
```

Comment:
Is this test check going to be uncommented in one of the further PRs?
Maybe another PR will introduce again this check with the new and right complexity check?

Reply:
This check is reintorodeuced with a different value in MIGENG-337

Reply:
Great, thanks @m-brophy